lib/services/utils/A6sRailwayUtil.ts

```diff
@@ -205,11 +205,19 @@ export class A6sRailwayUtil {
      * @return {Promise<IRailWayStation>}
      */
     async resolveTree(station: IRailWayStation, pwd: string, graphPath: string[] = []) {
+        const name = `${station.name}_${this.generateUniqueId()}`;
```

Comment:
Still don't like the approach when string concatenation is used. Can we define new type and use it inside the graph?